src/applications/disability-benefits/all-claims/pages/mentalHealthChanges.js

```diff
@@ -48,7 +50,7 @@ export const uiSchema = {
       'ui:title': 'Other',
     },
     otherExplanation: {
-      'ui:title': ' ',
+      'ui:title': explanationOfOther,
```

Comment:
Is there a reason it can't just have a regular label for everyone? It actually seems a little confusing to me that it lacks one, even though you can guess at it from the context of the question.

Reply:
@jbalboni : I have pinged UX for a reason and will get back to you when I know why.

Reply:
@jbalboni: Spoke with UX, they said the label for the box is "Other" right above it. They did want me to change the text of the hidden label though to be more specific. So I will make that change and ping you back when it's completed.

Reply:
@jeffbalboni: Spoke with UX again, they have changed their stance on hiding the label. This is what has been decided. 

![screen shot 2019-01-03 at 12 55 37 pm](https://user-images.githubusercontent.com/41440372/50653387-5d18f980-0f57-11e9-9c31-c7927bfcb080.png)

This change has been pushed up and ready for your review.